mkdocs-base.yml

```diff
@@ -164,20 +164,27 @@ nav:
       - Percona Everest 0.5.0 (2023-11-28): release-notes/Percona-Everest-0.5.0-(2023-11-28).md      
       - Percona Everest 0.4.0 (2023-10-30): release-notes/Percona-Everest-0.4.0-(2023-10-30).md      
   - Features: features.md
-  - Quickstart guide:
-      - quickstart-guide/quick-install.md
-      - quickstart-guide/eks.md
-      - quickstart-guide/gke.md
+  - Quickstart guide: quick-install.md
+    
   - Install:
-      - Setup and prerequisites: install/SetupPrereqs.md
-      - Install Percona Everest CLI: install/installEverestCLI.md
-      - Install Percona Everest: install/installEverest.md
-      - Uninstall Percona Everest: install/uninstallEverest.md 
-
+    - Setup and prerequisites:
+      - install/prerequisites.md
+      - install/supported_operators_k8s.md
+      - install/eks.md
+      - install/gke.md
+      - install/installEverestCLI.md
+    - install/installEverest.md
+    - install/install_everest_helm_charts.md
+  
   - Upgrade: 
-    - upgrade_with_cli.md
-    - upgrade_operators.md
-
+    - Upgrade Percona Everest using Helm: upgrade/upgrade_with_helm.md
+    - Upgrade Percona Everest using CLI: upgrade/upgrade_with_cli.md
+    - Upgrade Percona operators: upgrade/upgrade_operators.md
+
+  - Uninstall:
+    - Uninstall Percona Everest using CLI: uninstall/uninstallEverest.md
+    - Uninstall Percona Everest using Helm: uninstall/uninstall_everest_helm.md
```

Comment:
@rasika-chivate what do you think about changing `CLI` to `everestctl`? CLI is a generic term that means command line interface. Technically, helm is a CLI. To avoid ambiguity I suggest that we rename it to `everestctl` everywhere.

Reply:
I think it makes sense. Will change it everywhere.